powa/overview.py

```diff
@@ -80,11 +78,12 @@ def dashboard(self):
 
     @classmethod
     def get_childmenu(cls, handler, params):
+        from powa.server import ServerOverview
         children = []
         for s in list(handler.servers):
             new_params = params.copy()
             new_params["server"] = s[0]
             entry = ServerOverview.get_selfmenu(handler, new_params)
-            entry.title = s[1]
+            entry.title = handler.deparse_srvid(s[0])
```

Comment:
this adds an O(n) complexity for the breadcrumb.  You could use s[2] and add the necessary information in servers()@framework.py to have the information basically for free, ie:

```diff
diff --git a/powa/framework.py b/powa/framework.py
index 2e46bbc..28dcdfb 100644
--- a/powa/framework.py
+++ b/powa/framework.py
@@ -278,13 +278,14 @@ class BaseHandler(RequestHandler):
         """
         if self.current_user:
             if self._servers is None:
-                self._servers = [[s['id'], s['val']] for s in self.execute(
+                self._servers = [[s['id'], s['val'],s ['alias']] for s in self.execute(
                     """
                     SELECT s.id, CASE WHEN s.id = 0 THEN
                         %(default)s
                     ELSE
                         s.hostname || ':' || s.port
-                    END AS val
+                    END AS val,
+                    s.alias
                     FROM {powa}.powa_servers s
                     ORDER BY hostname
                     """, params={'default': self.current_connection})]
```

Reply:
Implemented via https://github.com/powa-team/powa-web/pull/176 instead.

Reply:
you should have told me you wanted to split that, I could have taken care of it.  Anyway I will merge it.